### PR TITLE
net-analyzer/goaccess: version bump, user and group

### PR DESCRIPTION
**Pull request by SpiderX**

### CODE REVIEW

File: net-analyzer/goaccess/goaccess-1.6.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+	diropts -o goaccess -g goaccess -m 0700
+	keepdir /var/lib/goaccess/db /var/log/goaccess
+}
+
+pkg_postinst() {
+	if ! has_version net-misc/geoipupdate ; then
+		einfo "You should consider to install net-misc/geoipupdate"
+		einfo "to be able to use GeoIP databases"
+	fi
```
Comment (juippis) [inline net-analyzer/goaccess/goaccess-1.6.2.ebuild:67]: Maybe switch this to use *optfeature.eclass*?

File: net-analyzer/goaccess/goaccess-1.6.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		"$(use_enable debug)" \
+		"$(use_enable geoip geoip "$(usex geoipv2 mmdb legacy)")" \
+		"$(use_enable unicode utf8)" \
+		"$(use_with getline)" \
+		"$(use_with ssl openssl)"
```
Comment (juippis) [inline net-analyzer/goaccess/goaccess-1.6.2.ebuild:48]: Maybe switch to the cleaner array form?

File: net-analyzer/goaccess/goaccess-1.6.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+SRC_URI="https://tar.goaccess.io/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="debug geoip geoipv2 getline ssl unicode"
+REQUIRED_USE="geoipv2? ( geoip )"
+
+RDEPEND="acct-user/goaccess
+	sys-libs/ncurses:=[unicode(+)?]
```
Comment (juippis) [inline net-analyzer/goaccess/goaccess-1.6.2.ebuild:19]: You're missing the dependency on group.

**Review by juippis — APPROVED** [review]
Looks good, thanks!

File: net-analyzer/goaccess/goaccess-1.6.2.ebuild (hunk 3 above)
Comment (SpiderX) [reply net-analyzer/goaccess/goaccess-1.6.2.ebuild:19]: @juippis I see that you added it. Do we really need it?
I have ACCT_USER_GROUPS sets and there is a call of acct-user_add_deps, which should do [this](https://github.com/gentoo/gentoo/blob/master/eclass/acct-user.eclass#L164).

Or there is some policy that requires include group in ebuild RDEPEND explicitly?

Comment (juippis) [reply net-analyzer/goaccess/goaccess-1.6.2.ebuild:19]: We generally don't rely on indirect deps. If something's needed by your package, and not directly inherited by an eclass, I'd say you're better listing it in the ebuild. IMHO it makes things consistent and easier to grasp in the ebuild too, and is future-proof too.

File: net-analyzer/goaccess/goaccess-1.6.2.ebuild (hunk 2 above)
Comment (SpiderX) [reply net-analyzer/goaccess/goaccess-1.6.2.ebuild:48]: @juippis 
Did you mean
```
        local myeconfargs=(
                "$(use_enable debug)"
                $(use_enable geoip geoip "$(usex geoipv2 mmdb legacy)")"
                ....
        )

        econf "${myeconfargs[@]}"
```

under array form?

File: net-analyzer/goaccess/goaccess-1.6.2.ebuild (hunk 3 above)
Comment (SpiderX) [reply net-analyzer/goaccess/goaccess-1.6.2.ebuild:19]: Got it. Thanks.

File: net-analyzer/goaccess/goaccess-1.6.2.ebuild (hunk 2 above)
Comment (juippis) [reply net-analyzer/goaccess/goaccess-1.6.2.ebuild:48]: Yes, and those quotation marks `"$(use_enable debug)"` should be unnecessary inside the array.

**Pull request closed without merging**